src/elements/content-sidebar/__tests__/SidebarNavSign.test.tsx

```diff
@@ -1,80 +1,75 @@
 import * as React from 'react';
-import { render, fireEvent } from '@testing-library/react';
+import { render, screen, fireEvent } from '@testing-library/react';
```

Comment:
we should use userEvent over fireEvent
we should also be importing these function from https://github.com/box/box-ui-elements/blob/master/src/test-utils/testing-library.tsx instead which unmock react-intl already